dockerfiles/run/guice/cassandra-ldap/destination/conf/elasticsearch.properties

```diff
@@ -34,6 +34,8 @@ elasticsearch.retryConnection.minDelay=3000
 elasticsearch.indexAttachments=true
 
 # Reports for metrics into ElasticSearch
+# Defaults to elasticsearch.masterHost : on which server to publish metrics
+elasticsearch.http.host=elasticsearch
```

Comment:
The metrics are published via http, and their config allow posting only to one host.

It's sad however : 
 - It is not mission critical
 - We do not expect high load (every 30s for each metric)